### ISSUE TITLE
Create Aktuelt section

### ISSUE THREAD

**Issue by niccofyren** [gathering/tgno-backend#7]
# General

The Aktuelt section is where our own articles (news) about The Gathering lives. These were the main content (apart from static info pages) of previous gathering.org versions. In current site they will mainly live in a separate section, but also be listed on frontpage and other locations when relevant.

# Endpoint

Should follow regular Wagtail REST Api setup and live under `.../news/`

# Data structure

It is probably quite safe to start with the structure we know from existing page. Once frontend is in place we can adjust as needed which fields are shown, and how they are used (or removed).

Fields we should expect in first version
- Body
- Intro
- Title
- Main image
- Contributors (some could be writers, other could be illustrators or photographers)
- Tags
- Publishing fields (we should get these from Wagtail out of the box, think last edited, scheduling, etc)

Fields we should avoid for now
- Side column items* (facts, menu, related content, ...)

\* New page is intended to be less hands-on. Meaning we want to expose less of these type of options to users initially, and instead rely on fixed logic for what shows up and where.

# Presentation

Out of scope in this initial backend implementation. But in short it will probably be
- A fixed grid based view, similar to https://archive.gathering.org/tg19/ (just without it being manually configurable by users)
- List all articles/news in chronological order, with pagination
- Other relevant widgets/items might show up in grid
- Articles/news might show up in other sections, where relevant
- Each article will allow easy access to related articles and information pages
